src/stcal/jump/jump.py

```diff
@@ -129,33 +129,45 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
     # Multiprocessing has been disabled for now, so the nunber of slices
     # is here set to 1. I'm leaving the related code in to ease the eventual
     # re-enablement of this code.
-    n_slices = 1
-
-    yinc = int(n_rows / n_slices)
+    print('new stcal jump')
```

Comment:
can't have print statements in production code. Either remove or replace with log messages.